pom.xml

```diff
@@ -725,11 +725,6 @@
                 <artifactId>joda-time</artifactId>
                 <version>2.12.7</version>
             </dependency>
-            <dependency>
-                <groupId>junit</groupId>
-                <artifactId>junit</artifactId>
-                <version>4.13.2</version>
-            </dependency>
```

Comment:
Removing the entry from dependency management does not remove the dependency.  All it does is to remove our control over the version of the dependency that will be used.  It's not a fault in the dependency updating tool that it is adding this entry back in; the tool is effectively telling you that the dependency is not gone.

Looking at the project I can see that it is coming in as a transitive dependency of `json-simple`, which is referenced by the `job-service-acceptance-tests` project:

![image](https://github.com/JobService/job-service/assets/20627297/30672de1-9c9d-45be-8468-094061373803)

Incidentally I had thought that we were no longer using `json-simple` following the [`US915106`](https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=915106) changes that I made about a month ago.  Obviously I was relying on the dependency reports which are not reporting on the test usages.